### PR TITLE
Use double quotes in SSH command.

### PR DESCRIPTION
**Pull request by XhmikosR**
This works on Windows too.

@ostark: I tested only on Windows, so please confirm before merging.

```shell
xmr@xmr-PC MSYS /c/Users/xmr/Desktop/foo
$ ./craft copy/db/down

Export DB remote - import locally: foostaging
---------------------------------------------------

 Are you sure? (yes/no) [yes]:
 >

Dump imported
====================================================================== 100 %
time:  11 secs/11 secs

Performed steps:
----------------

 * Creating dump on remote (./storage/craft-copy-transfer.sql)
 * Downloading dump from remote ./storage/craft-copy-transfer.sql
 * Creating backup of local DB (./storage/craft-copy-recent.sql)
 * Importing dump

Rollback?
---------

php craft copy/db/from-file ./storage/craft-copy-recent.sql

```

Refs #4

### CODE REVIEW

**Comment by XhmikosR** [comment]
BTW I suspect it's a similar issue for the other failure I have on Windows when syncing the assets, but I couldn't fix it from a quick look.

Happy to test if you guys come up with a patch.

File: src/services/Ssh.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ public function upload($src, $target)
      */
     public function download($src, $target)
     {
-        $cmd = "ssh {$this->remote} 'cat {$src} | gzip' | zcat > {$target}";
+        $cmd = "ssh {$this->remote} \"cat {$src} | gzip | zcat > {$target}\"";
```
Comment (ostark) [inline src/services/Ssh.php]: Did you change the wrapping intentionally? IMO it's a mistake.
Only the `cat {$src} | gzip` part should be executed on the remote.

Comment (XhmikosR) [reply src/services/Ssh.php]: Oops, that was a mistake, thanks for catching it!

Comment (XhmikosR) [reply src/services/Ssh.php]: Fixed.

**Comment by XhmikosR** [comment]
Wait, this broke again...

**Comment by XhmikosR** [comment]
@ostark: so, yeah, it seems it doesn't work for me with the quotes after `gzip`.

**Comment by ostark** [comment]
What about?:

```
# Ssh::upload()

$cmd =  sprintf('cat "%s" | gzip | ssh %s "zcat > %s"', $src, $this->remote, $target);
```


```
# Ssh::download()

$cmd = sprintf('ssh %s "cat %s | gzip" | zcat > "%s"', $this->remote, $src, $target);
```

**Comment by XhmikosR** [comment]
Still fails for me on Windows with the same error that `zcat` isn't found.

**Comment by ostark** [comment]
is `gzcat`, `zcat` or `gunzip` available?

**Comment by XhmikosR** [comment]
`gzcat` no. The other two yes.

```
xmr@xmr-PC MSYS ~
$ which zcat
/usr/bin/zcat

xmr@xmr-PC MSYS ~
$ which gunzip
/usr/bin/gunzip
```

**Comment by XhmikosR** [comment]
OK, I think I'm gonna close this since it changes the logic.

But at least we pinpointed the point of failures.

**Comment by XhmikosR** [comment]
It's so weird because like the rsync command if I just run it via MSYS2 it works just fine.

**Pull request closed without merging**